packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -141,6 +143,19 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         this.setState({ collapsedNodes: newList });
     }
 
+    private eventClicked(timeStampObj: { [key: string]: number }) {
+        const offset = this.props.viewRange.getOffset();
+        const rawTimeStamp = timeStampObj['timeStamp'];
+        const newViewRangeStart = rawTimeStamp - (offset ? offset : 0);
+        this.props.unitController.selectionRange = {
+            start: newViewRangeStart,
+            end: newViewRangeStart
+        };
+        this.chartLayer.setEventClickedFlag(true);
```

Comment:
I would prefer to separate the timestamp sync and row sync in 2 different pull requests (in theia-trace-extesion and timeline-chart).